consent_manager/api.rst

```diff
@@ -252,6 +252,35 @@ Code::
 
     :param string|object error: **Required** Error code or exception.
 
+Send initial stats
+`````````````````````````````````````
+Command send initial stats (no decision) to Consent Manager collector.
```

Comment:
Command send**s** initial stats (no decision) to Consent Manager **stats** collector.